tests/test_deprecation.py

```diff
@@ -1,9 +1,7 @@
+from manim.utils.deprecation import deprecated, deprecated_params
 import warnings
 
-import pytest
-
-from manim import logger
-from manim.utils.deprecation import deprecated, deprecated_params
+warnings.filterwarnings("ignore")
```

Comment:
why do you need this?

Reply:
Every time I run pytest it gives me every warning the depreciation test spits out. that disables that.

Reply:
huh, why doesn't it happen in CI?

Reply:
I can't tell, I've asked on discord. It seems to only happen to me.